documentation/components/navigation/_navigation.pug

```diff
@@ -0,0 +1,14 @@
+mixin makeNavUL(items)
+  ul.nav
+    each val, index in items 
+      li
+        - if (typeof(val) === 'array') {
```

Comment:
you don't need the dash `-` and javascript for conditionals, [pug has conditionals by default](https://pugjs.org/language/conditionals.html)

Reply:
Yeah my bad there. I clearly haven't spent much time with Pug recently.